doc/classes/Directory.xml

```diff
@@ -85,6 +85,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the current drive index of the currently opened directory. See [method get_drive] to convert returned index to the name of the drive.
```

Comment:
Try to avoid "of the" constructions in favor of possessives:

    Returns the currently opened directory's drive index.